README.md

```diff
@@ -8,7 +8,8 @@ We officially began work on the new fork on January 21st, 2021.  Since then, we'
 
 ## Getting Started
 
-If you would like to install or run this project please see the [Developer Guide](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md). There will be binary distributions coming soon which will enable faster installations. 
+If you would like to use a binary, unzip and run this project, please see the [Downloads Page](https://opensearch.org/downloads.html).
```

Comment:
We should mention the docker container.

Reply:
If you would like to install and run this project please see the [Downloads Page](https://opensearch.org/downloads.html).

Something like that could work too. Options are also enumerated in the downloads page.

Reply:
Sure.

Reply:
I like that. I'll leave it open as @elfisher suggested.